### PR TITLE
Platform: Qdrant destination connector

### PR DESCRIPTION
**Pull request by Paul-Cornell**
This is now publicly available.

See https://unstructured-53-qdrant-platform-2024-12-13.mintlify.app/platform/destinations/qdrant

### CODE REVIEW

File: snippets/general-shared-text/qdrant.mdx [hunk 1]
```diff
@@ -1,6 +1,10 @@
-- For [Qdrant local](https://github.com/qdrant/qdrant), the path to the local Qdrant installation, for example: `/qdrant/local`
-- For [Qdrant client-server](https://qdrant.tech/documentation/quickstart/), the Qdrant server URL, for example: `http://localhost:6333`
-- For [Qdrant Cloud](https://qdrant.tech/documentation/cloud-intro/):
+- For the [Unstructured Platform](/platform/overview), only [Qdrant Cloud](https://qdrant.tech/documentation/cloud-intro/) is supported.
+- For [Unstructured Ingest](/ingestion/overview), Qdrant Cloud, 
+  [Qdrant local](https://github.com/qdrant/qdrant), and [Qdrant client-server](https://qdrant.tech/documentation/quickstart/) are supported.
```
Comment (MKhalusova) [inline snippets/general-shared-text/qdrant.mdx:3]: How do we support Qdrant local in SaaS platform? I don't understand. We probably should clarify this.

**Review by MKhalusova — COMMENTED** [review]
Looks great! There's one minor thing I think we should probably clarify. See my inline comment.

File: snippets/general-shared-text/qdrant.mdx (hunk 1 above)
Comment (Paul-Cornell) [reply snippets/general-shared-text/qdrant.mdx:3]: By SaaS platform, do you mean the Unstructured Platform? If so, the above states that only Qdrant Cloud is supported.

Comment (MKhalusova) [reply snippets/general-shared-text/qdrant.mdx:3]: My bad, missed that.

**Review by MKhalusova — APPROVED** [review]

**Pull request merged**